packages/core/apps/core-apps-server-internal/src/bundle_routes/register_bundle_routes.ts

```diff
@@ -7,10 +7,10 @@
  */
 
 import { join } from 'path';
-import { PackageInfo } from '@kbn/config';
+import type { PackageInfo } from '@kbn/config';
 import { fromRoot } from '@kbn/utils';
 import UiSharedDepsNpm from '@kbn/ui-shared-deps-npm';
-import * as UiSharedDepsSrc from '@kbn/ui-shared-deps-src';
+import { distDir as UiSharedDepsSrcDistDir } from '@kbn/ui-shared-deps-src';
```

Comment:
`import * as ...` doesn't work in packages. I took the approach of renaming the specific dependency.